libs/embed/src/embed.ts

```diff
@@ -196,6 +196,32 @@ class Novu {
   initializeIframe = (clientId: string, options: any) => {
     if (!document.getElementById(IFRAME_ID)) {
       const iframe = document.createElement('iframe');
+      window.addEventListener(
```

Comment:
Add the `message` event listener right after the `iframe` DOM element is created. This way the `Embed` will never miss the events. Before we were doing it in the `onload` which is too late for some browsers.